### ISSUE TITLE
docs: GitHub-Actions.md

### ISSUE THREAD

**Issue by csharpee** [jekyll/jekyll#8360]
The page referring to is here: https://jekyllrb.com/docs/continuous-integration/github-actions/

## Motivation

After following instructions on the github-actions.md page, it seems the instructions are clear, but do not work with a GitHub Public Organization repository making use of GitHub Pages & Actions.

## Suggestion

Improve the instructions and update the page if the GH-Pages and Jekyll 4.0 are unable to work together harmoniously. I am no expert with GH Actions, but see it may be possible to get it working with Jekyll, although the default setup on the github-actions.md page do not produce successful results for a GH public organization on pushing to master locally.

I also opened a topic here to hopefully receive some guidance on resolving this issue. 
https://github.community/t/jekyll-4-0-github-pages-and-actions/129397

**Comment by csharpee** [comment]
On following up, I decided to run a simple easy to test solution that has been proven to work. I used a fork of this repository for my own organizational account. What is it that I am doing wrong? Is it me or is it GitHub? Only thing I can think of is the difference of an ORG account and a USERNAME, if so, how do I know this is fact?

https://github.com/MichaelCurrin/jekyll-actions-quickstart/issues/3

You can find the results here: 
https://github.com/Galactic-Society/jekyll-actions-quickstart/actions

Logs again, here: 
```
error: src refspec ***@github.com/Galactic-Society/jekyll-actions-quickstart.git does not match any
error: failed to push some refs to 'https://'
```

**Comment by csharpee** [comment]
In the instructions on the jekyll core website, maybe add a tip saying "after github actions is setup with jekyll-actions and the new gh-pages branch is created successfully, remember to switch from master to the **gh-pages** in your account repository. 

For me to get this to work as stated on the website, I had to recreate an org. name and fork the repo to new org. account.

Only jekyll-actions 2.0.1 worked for me; I tested 2.0.2+ and all of them failed to build to master branch. 
After gh-pages branch is created, switch to gh-pages branch via account settings for github pages.

Further reading,
https://github.com/forestryio/jekyll-menus/issues/19#issuecomment-683131856
